client/components/Chat.vue

```diff
@@ -118,6 +129,24 @@ export default {
 		hideUserVisibleError() {
 			this.$root.currentUserVisibleError = null;
 		},
+		editTopic() {
+			this.channel.editTopic = true;
+
+			this.$nextTick(() => {
+				$(`#chan-${this.channel.id} .topic-edit`).focus();
+			});
+		},
+		saveTopic(event) {
+			this.channel.editTopic = false;
+			const newTopic = event.target.value;
+
+			if (newTopic && this.channel.topic !== newTopic) {
```

Comment:
This check prevents you from removing a topic.

Reply:
I actually added that check after I noticed that sending only `/topic` just prints out the current topic in the channel. I could not find a standard way of clearing topic in IRC. However, I could set the topic equal to one white-space character if the user sends an empty topic. What do you think?

Reply:
That's a good point. Leave it as is then.

Reply:
Edit, apparently it's possible by doing `> TOPIC #ircv3 :`, but that also requires changing the server command to work with this.

Reply:
I created #3351 to handle this

Reply:
Can you change this to send `/raw TOPIC ${channelname} :${newTopic}` command?

I just don't know if `/topic ` is a good idea to clear the topic. #3352 fixes the raw command so the colon works as expected.

Reply:
I changed this to use raw command and tested that it works as expected with your change.